File: 383-ransom-note/383-ransom-note.go
func canConstruct(ransomNote string, magazine string) bool {
    
    
    box := make([]int ,26)
    for _,v := range magazine{
        box[v-'a'] ++
    }
    for _,v := range ransomNote{
        box[v-'a']--
        if box[v-'a'] < 0 {
            return false 
        }
    }
    return true 

}

File: 383-ransom-note/383-ransom-note.py
class Solution:
    def canConstruct(self, ransomNote: str, magazine: str) -> bool:

        
        mcntr = Counter(magazine) 

        for e in ransomNote: 
            if mcntr[e] ==0:
                return False 
            mcntr[e] -=1 
        return True 


    
